README.md

```diff
@@ -1,10 +1,93 @@
 # Aind.Behavior.Pirouette
 
-A repository with source code for long-term, freely moving, electrophisiology recordings in mice
+Authors: Bruno Cruz, Brandon Pratt, and Carl Schoonover
 
-## Getting started
+Last Updated: 05/15/2025
 
-1. Clone this repository
-2. Regenerate the Bonsai environment by running `setup.ps1` from the `./bonsai` directory
-3. Open Bonsai by running `./bonsai/Bonsai.exe`
-4. Open the desired script from the `./src` directory
\ No newline at end of file
+A repository with source code and instructions for long-term, electrophisiology recordings in behaving mice
+
+---
+## Content
+### 1. Getting started 
+### 2. Pirouette Bonsai workflow
+### 3. Lifealert 
+### 4. Neural activity visualization
+---
+### 1. Getting started 
```

Comment:
This sounds like instructions to setup the experiment in the Allen. As we discussed previously, it would be nice to leave the entry-point to the repo somewhat agnostic. Wouldnt it make more sense to put these instructions in a separate docs file? We can also just trigger the build of the sphinx website and move this there.

You should also consider linking, or reproducing the following deployment scipt: https://github.com/AllenNeuralDynamics/Aind.Behavior.Services/blob/main/scripts/install_dependencies.ps1

This is what I use for all tasks and it is what SIPE is using as reference for automatic deployment